### ISSUE TITLE
'abvf' example does not match description

### ISSUE THREAD

**Issue by dscorbett** [simoncozens/feature-tags#1]
The description for 'abvf' explains how U+17BE KHMER VOWEL SIGN OE is split into a top part and a left part. However, the rendered example only includes the top part.
<img width="357" alt="On: យល់ឃើញ" src="https://user-images.githubusercontent.com/1124347/128080881-b04d3b11-5ecb-4304-bc39-a6c575004971.png">
The original Noto Sans Khmer does include the left part though. Maybe something went wrong in the subsetter.
![យល់ឃើញ](https://user-images.githubusercontent.com/1124347/128081281-613f6024-acaa-487a-a54a-8fde410e101f.png)

**Comment by simoncozens** [comment]
You're right, and something went wrong with the subsetter. That's a very tricky one to fix.

**Comment by simoncozens** [comment]
Filed as https://github.com/fonttools/fonttools/issues/2387
